ios/keyman/Keyman/resources/OfflineHelp.bundle/Contents/Resources/fonts.css

```diff
@@ -0,0 +1,139 @@
+@font-face {
```

Comment:
Note, these fonts rely on external s.keyman.com. Is it intentional to keep them that way?

Reply:
In reality, we shouldn't be using fonts.css ...

Reply:
I'd say it's the latter - probably shouldn't be using it.  And, since the fonts aren't even available when offline... we basically aren't.  The offlining script just doesn't have a filter set to fully block it, and something picks it up during the scraping process.

Actually, from what I can tell, the actual .html pages don't link with it.  If anything, the issue is that they do make requests against https://fonts.googleapis.com that silently fail.

Either way, I've pulled up the results in-app with device wi-fi off, and everything displays fine.